src/interceptors/response.ts

```diff
@@ -215,8 +226,17 @@ export function defaultResponseInterceptor(
       throw error;
     }
 
+    if (!isMethodIn(config.method, cacheConfig.methods)) {
+      if (__ACI_DEV__) {
+        axios.debug?.({
+          msg: `Ignored because method (${config.method}) is not in cache.methods (${cacheConfig.methods})`
```

Comment:
```suggestion
          id: config.id,
          msg: `Ignored because method (${config.method}) is not in cache.methods (${cacheConfig.methods})`
          data: { config, cacheConfig }
```

Adds more information in case someone needs it.

Reply:
I cannot add a suggestion, but seems that other debug calls below here are missing the `id` parameter too.

Reply:
I added `id` to all expect for the one block that actually checks for its existence. I think we are good.